tools/validate_configs/test_configs/hpc-cluster-small-slurm-v5.yaml

```diff
@@ -68,6 +68,10 @@ deployment_groups:
     - compute_node_group
     settings:
       partition_name: compute
+      startup_script: |
```

Comment:
Could we do something similar in the controler? We previously had a regression there.

Reply:
Also it would be better to test with the actual startup script module as the startup script module output is not known at apply time which can reveal other issues.

Reply:
As you pointed out offline, we might want to include this in one of our actual integration tests to make sure terraform plan is being called.

Reply:
I updated the test_config and ran the test manually. 
As we discussed offline, it will be best to add the test_config to the integration tests in a separate PR. The v5 tests are all based on examples currently, and adding dummy startup-script modules to those tests would be fragile. Adding a new test based on this test_config will be a cleaner way of incorporating startup-scripts.